org.bzflag.BZFlag.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "org.bzflag.BZFlag",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "bzfs",
+  "rename-icon": "bzflag",
+  "rename-desktop-file": "bzflag.desktop",
+  "finish-args": [
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11",
+    "--device=dri",
+    "--share=network"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.0.0.json",
+    "shared-modules/glew/glew.json",
+    {
+      "name": "c-ares",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://c-ares.haxx.se/download/c-ares-1.14.0.tar.gz",
+          "sha256": "45d3c1fd29263ceec2afc8ff9cd06d5f8f889636eb4e80ce3cc7f0eaf7aadc6e"
+        }
+      ]
+    },
+    {
+      "name": "bzflag",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.bzflag.org/bzflag/source/2.4.14/bzflag-2.4.14.tar.gz",
+          "sha256": "80089192c39416b73b6dd3fa7f48775f6d7453f001e347a5afc1b5c52fbe5611"
+        },
+        {
+          "type": "file",
+          "path": "org.bzflag.BZFlag.appdata.xml"
+        }
+      ],
+      "post-install": [
+        "install -Dm644 data/bzflag.desktop /app/share/applications/bzflag.desktop",
+        "install -Dm644 data/red_icon.png /app/share/icons/hicolor/64x64/apps/bzflag.png",
+        "install -Dm644 org.bzflag.BZFlag.appdata.xml /app/share/appdata/org.bzflag.BZFlag.appdata.xml"
```

Comment:
`"install -Dm644 -t /app/share/metainfo org.bzflag.BZFlag.appdata.xml"`

Reply:
I'm not quite sure what you mean by this suggestions.

Reply:
This is just a minor thing, but by using `-t` you don't have to repeat the (long) filename.